test/SwooleRequestHandlerRunnerTest.php

```diff
@@ -78,7 +80,13 @@ public function testConstructorRaisesExceptionWhenManagerPidIsNotZero(): void
 
     public function testRunRegistersExpectedHttpServerListenersAndStartsServerWhenInBaseMode(): void
     {
-        $this->httpServer->mode = SWOOLE_BASE;
+        if (PHP_VERSION_ID >= 80300) {
+            $reflection = new ReflectionProperty($this->httpServer, 'mode');
+            /** @psalm-suppress UnusedMethodCall */
+            $reflection->setAccessible(true);
```

Comment:
This is unnecessary since 8.1
https://www.php.net/manual/en/reflectionproperty.setaccessible.php

Reply:
Thank you, I wasn't aware of that